src/main/java/fr/greencodeinitiative/java/JavaEcoCodeProfile.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * ecoCode - Java language - Provides rules to reduce the environmental footprint of your Java programs
+ * Copyright © 2023 Green Code Initiative (https://www.ecocode.io)
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+package fr.greencodeinitiative.java;
+
+import org.sonar.api.server.profile.BuiltInQualityProfilesDefinition;
+import org.sonarsource.analyzer.commons.BuiltInQualityProfileJsonLoader;
+
+import static fr.greencodeinitiative.java.JavaRulesDefinition.LANGUAGE;
+import static fr.greencodeinitiative.java.JavaRulesDefinition.REPOSITORY_KEY;
+
+public final class JavaEcoCodeProfile implements BuiltInQualityProfilesDefinition {
```

Comment:
when is this profile created ? automatically, when the developer will launch unit tests ?
what about conflicts with developers using docker method ?

Reply:
We should not force the change of default profile, but simply propose a profile grouping all the ecoCode rules in order to facilitate their application for those who might be interested.
The presence of this profile will help with automated integration tests

Reply:
This profile is automatically created when plugin is installed.